README.md

```diff
@@ -1,2 +1,217 @@
-# Microverse_hello
-It is about a project which introduces some skills about how to use linters in html and css 
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+ 
+  <br/>
+
+  <h3><b>Microverse</b></h3>
```

Comment:
[OPTIONAL] Please always customize all content of your README to present your project even if you use a template you can mention the owner in the acknowledgement section but the title should be that of your project